### DISCUSSION TITLE
Missing letters

### DISCUSSION THREAD

**Discussion by dmifer** [mhammond/pywin32#2541] [General]
I wrote the script for reading emails and downloading attachments on everyday basis (50+ emails each day).

In general, it's working properly, but sometimes script skips the emails without any reason. Why it might happen?

I use application connection
```python
outlook = win32com.client.Dispatch("Outlook.Application").GetNamespace("MAPI")
```

and emails extracting from specific folder (I am sure that subject of skipped emails contain ```test```): 
```python
main_box = outlook.Folders.Item(main_folder)
inbox = main_box.Folders.Item(folder)
messages = inbox.Items
messages = [msg for msg in messages if "test" in msg.Subject.lower()]
```

**Comment by mhammond** [comment]
This is a support request. Please see README.md or the text you deleted when opening this issue.
